flang/runtime/command.cpp

```diff
@@ -14,6 +14,7 @@
 #include "flang/Runtime/descriptor.h"
 #include <cstdlib>
 #include <limits>
+#include <time.h>
```

Comment:
We should use the C++ headers for this (i.e. `<ctime>` in this case)
This will mean prefixing all the C functions with `std::`, e.g. `std::strchr`, `std::time_t`, `std::time` etc